src/app/pages/ProfileScreen/ProfileScreen.tsx

```diff
@@ -18,6 +21,7 @@ class ProfileScreen extends PureComponent<ProfileProps> {
         <ProfileTitle>
           {intl.formatMessage({ id: 'profile.title' })}
         </ProfileTitle>
+        <Text>{Config.TYPE}</Text>
```

Comment:
what the purpose of this?

Reply:
to make sure we are loading proper env file, everytime doing some changes in build process and CI process i need to add such small piece of code...Its a starter not a final project so its good to have information that proper env type is loaded.